jazz-ui/src/app/secondary-components/create-service/internal/create-service.component.html

```diff
@@ -487,7 +487,7 @@ <h4 class="bold-title">
                 <div class="row">
                     <div class="col-lg-4 col-md-4">
                         <p class="step-title">AWS Events</p>
-                        <p class="step-details">Select appropriate events that you want to subscribe to.</p>
+                        <p class="step-details">Select appropriate events that you want to subscribe to. Only tmodevops AWS account is supported at this point*</p>
```

Comment:
Can we remove the reference to internal here?

Reply:
Made this config-driven